notificationworkerlambda/src/main/scala/com/gu/notifications/worker/delivery/fcm/FcmClient.scala

```diff
@@ -99,7 +99,7 @@ class FcmClient (firebaseMessaging: FirebaseMessaging, firebaseApp: FirebaseApp,
       import FirebaseHelpers._
       val start = Instant.now
       firebaseMessaging
-        .sendMulticastAsync(message)
+        .sendEachForMulticastAsync(message)
         .asScala
         .onComplete { response =>
```

Comment:
It looks like we are only logging out in the case where the future completed successfully. If we're concerned about performance implications would it be worth logging an error on failure here to help track if we are experiencing errors? 
May be overkill, but could potentially introduce an alert if number of failures are of a certain level

Reply:
yeah, we do.  The input argument `response` for the `onComplete` call is actually a `Try[BatchResponse]` object.  We handle errors from the API call [here](https://github.com/guardian/mobile-n10n/blob/73ba5e726cc63ec5355f6ce5479dff5b4a9fb13f/notificationworkerlambda/src/main/scala/com/gu/notifications/worker/delivery/fcm/FcmClient.scala#L147).  When the API succeeds but only some of the tokens fail, we handle the errors [here](https://github.com/guardian/mobile-n10n/blob/73ba5e726cc63ec5355f6ce5479dff5b4a9fb13f/notificationworkerlambda/src/main/scala/com/gu/notifications/worker/delivery/fcm/FcmClient.scala#L120)

Reply:
Ah gotcha. Looks good